packages/pattern-picker/src/item.tsx

```diff
@@ -11,23 +12,39 @@ type Props = {
 	pattern: Pattern;
 };
 
+const getPatternMetaValue = ( patternMeta: string, regexp: RegExp ) => {
+	const [ , value ] = patternMeta.split( regexp );
+	if ( value ) {
+		return value.replace( /%2C/g, ',' );
+	}
+
+	return null;
+};
```

Comment:
I would make this function `parsePatternMetaData` and parse the whole data into a key value object. Having regexes outside this function is kind of leaky.

Reply:
Okay, that makes sense to me!

Reply:
I've made a new function `parsePatternMetaData` and moved this one to the internal function. What do you think?